src/app/cases/page.tsx

```diff
@@ -5,27 +5,23 @@ import { UUID } from 'crypto';
 import { CaseListing } from '../../types/schema';
 import { getNCases } from '../../api/supabase/queries/cases';
 import ListingCard from '../../components/ListingCard/ListingCard';
-import {
-  CardColumn,
-  CaseDetailDisplay,
-  CaseDetails,
-  MainDisplay,
-  PageContainer,
-} from './styles';
-import { H1, H2 } from '../../styles/text';
+import CaseDetailDisplay from '../../components/CaseDetails';
+import { CardColumn, MainDisplay, PageContainer } from './styles';
+import { H1 } from '../../styles/text';
```

Comment:
I think we should refactor to use `@/` to make things cleaner, though the changes in main branch likely has that (make sure to double check the imports are still right!)